samples/generate_html5_console.py

```diff
@@ -69,11 +69,15 @@ def main():
 
     args = get_args()
 
+    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+    context.verify_mode = ssl.CERT_NONE
+
     try:
         si = SmartConnect(host=args.host,
```

Comment:
SmartConnectNoSSL is an alternative if you want to skip the cert check. You don't need to create the context if you use that.